Workade/Views&ViewModels/Magazine/TipDetail/TipDetailCell/TipDetailCell.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  TipDetailCell.swift
+//  Workade
+//
+//  Created by Hong jeongmin on 2022/10/21.
+//
+
+import UIKit
+
+class TipDetailCell: UICollectionViewCell {
+    let titleLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .theme.background
+        label.font = .customFont(for: .headline)
+        label.numberOfLines = 0
+        label.translatesAutoresizingMaskIntoConstraints = false
+        
+        return label
+    }()
+    
+    let bookMarkButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(systemName: "bookmark"), for: .normal)
```

Comment:
```suggestion
        let config = UIImage.SymbolConfiguration(pointSize: 17, weight: .semibold)
        button.setImage(UIImage(systemName: "bookmark", withConfiguration: config), for: .normal)
```

[P1] 북마크 버튼의 굵기를 반영시켰습니다!

Reply:
반영 하였습니다 :)